src/main/java/net/sf/jabref/bibtex/EntrySorter.java

```diff
@@ -127,18 +117,14 @@ public void databaseChanged(DatabaseChangeEvent e) {
                 break;
             case CHANGED_ENTRY:
                 // Entry changed. Resort list:
-                //Collections.sort(set, comp);
```

Comment:
Should the `resort list` comment be removed, too? Or is the implementation below some kind of resorting?!